hs_core/management/commands/migrate_netcdf_resources.py

```diff
@@ -0,0 +1,132 @@
+import os
+import logging
+
+from django.core.management.base import BaseCommand
+
+from hs_core.hydroshare import current_site_url, set_dirty_bag_flag
+from hs_core.models import ResourceFile, CoreMetaData, Coverage
+from hs_core.views.utils import rename_irods_file_or_folder_in_django
+from hs_app_netCDF.models import NetcdfResource
+from hs_file_types.models import NetCDFLogicalFile
+from ..utils import migrate_core_meta_elements
+
+
+class Command(BaseCommand):
+    help = "Convert all multidimensional resources to composite resource"
+
+    def handle(self, *args, **options):
+        logger = logging.getLogger(__name__)
+        resource_counter = 0
+        to_resource_type = 'CompositeResource'
+        msg = "THERE ARE CURRENTLY {} MULTIDIMENSIONAL RESOURCES PRIOR TO CONVERSION.".format(
+            NetcdfResource.objects.all().count())
+        logger.info(msg)
+        print(">> {}".format(msg))
+
+        for nc_res in NetcdfResource.objects.all():
+            # get the nc file name which needs to be used to create a new folder
```

Comment:
It'd be helpful to wrap each item with a try/except block with a message if it fails.  Right now, if something fails, it blocks the whole thing.

Reply:
Have added error checking for potential errors.